src/ORM/Behavior/Translate/ShadowTableStrategy.php

```diff
@@ -525,7 +530,7 @@ protected function rowMapper(CollectionInterface $results, string $locale): Coll
 
             if ($hydrated) {
```

Comment:
Here too `instanceof EntityInterface` check is missing for the existing code, then the `@var` annotation can also be removed.